ci-operator/step-registry/ipi/install/install/ipi-install-install-commands.sh

```diff
@@ -101,6 +109,11 @@ ret="$?"
 
 echo "$(date +%s)" > "${SHARED_DIR}/TEST_TIME_INSTALL_END"
 
+START_TIME=$(cat $SHARED_DIR/TEST_TIME_INSTALL_START)
```

Comment:
This is not the start time that we want to use. We want the time right before calling `openshift-install create cluster`.